mdata/cache/accnt/lru.go

```diff
@@ -0,0 +1,43 @@
+package cache
+
+import (
+	"container/list"
+)
+
+type entry struct {
```

Comment:
I see the value now of keeping the lru more abstract. it means that in the unit tests you can work with strings instead of EvictTargets. that's kindof nice.

BTW `type entry` is still there, seems unused.